### PR TITLE
update my keymap for Infinity Ergodox

### PR DESCRIPTION
**Pull request by dudeofawesome**
## Description

Just updating my keymap for Infinity Ergodox.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

### CODE REVIEW

File: keyboards/ergodox_infinity/keymaps/dudeofawesome/keymap.c [hunk 1]
```diff
@@ -2,19 +2,7 @@
 #include "debug.h"
 #include "action_layer.h"
```
Comment (noroadsleft) [inline keyboards/ergodox_infinity/keymaps/dudeofawesome/keymap.c]: Neither of these lines should be needed. QMK_KEYBOARD_H includes both of these files.

Comment (dudeofawesome) [reply keyboards/ergodox_infinity/keymaps/dudeofawesome/keymap.c]: Hmm, interesting. I think that's from the input club keymap that I based mine off of. 
I'll try removing them.

Comment (dudeofawesome) [reply keyboards/ergodox_infinity/keymaps/dudeofawesome/keymap.c]: works fine with imports removed!

**Comment by noroadsleft** [comment]
Looks good, thanks!

**Pull request merged**